apps/monitor-client/lib/features/users-view/components/UsersView.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react';
+import { GridColDef, DataGrid } from '@mui/x-data-grid';
+import { Grid } from '@mui/material';
+import { styled } from '@mui/material/styles';
+import { useFetchContacts, useFetchUsers } from '../hooks';
+
+const StyledDataGrid = styled(DataGrid)({
+    fontFamily: 'monospace',
+    fontSize: '12px',
+    background: '#FFF',
+    height: '400px',
+});
+
+export function UsersView() {
+    const {
+        usersData,
+        isUsersLoading,
+        isUsersError,
+        usersPage,
+        setUsersPage,
+        usersPageSize,
+        setUsersPageSize,
```

Comment:
I think we can change entity-centric names to common:
const { data, isLoading, isError, page, setPage, pageSize, setPageSize } = useFetchUsers();

The same approach is for other hooks. And this changes should be applied for hook itself

Reply:
but how we should manage data from different hooks among one component. we should somehow recognize data is got from different hooks

Reply:
do we have such a component?

Reply:
even if we have this will be resolved by the import